Dockerfile

```
@@ -1,2 +1,3 @@
 FROM islasgeci/base:latest
 COPY . /workdir
+RUN Rscript -e "install.packages(c('covr','DT', 'rjson', 'styler'), repos='http://cran.rstudio.com')"
```

Las librerías `covr`, `DT` y `styler` ya vienen en la imagen base, así que no es necesario que las volvamos a intalar.

Por favor borra esas paqueterías de la línea 3:
```suggestion
RUN Rscript -e "install.packages('rjson', repos='http://cran.rstudio.com')"
```